### ISSUE TITLE
v1alpha3 planning

### ISSUE THREAD

**Issue by ffromani** [k8stopologyawareschedwg/noderesourcetopology-api#27]
Once https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/issues/24 is done, do we want or need another v1alpha version or should we jump to v1beta1?

This issue is meant to track the requirements and the planning for `v1alpha3`

**Comment by swatisehgal** [comment]
If we identify items down the line that justify the need for `v1alpha3`we can move to the next alpha version but at this point, I think we should directly aim to move to v1beta1.

**Comment by ffromani** [comment]
> If we identify items down the line that justify the need for `v1alpha3`we can move to the next alpha version but at this point, I think we should directly aim to move to v1beta1.

I fully agree. I filed this issue mostly for due diligence, but I'm also convinced `v1beta1` is the next logical step.

**Comment by fmuyassarov** [comment]
@ffromani are you planning to cut a release for v1alpha2?
